### PR TITLE
Add in ui and enable multi-db support of operations

### PR DESCRIPTION
**Pull request by Chris7**
Adds cli interface for working with operations, adds in a unique uuid per database for scoping changes to a given db

### CODE REVIEW

File: migrations/01-initial/up.sql [hunk 1]
```diff
@@ -93,3 +97,10 @@ CREATE TABLE block_group_edges (
 CREATE UNIQUE INDEX block_group_edges_uidx ON block_group_edges(block_group_id, edge_id);
 
 INSERT INTO sequence (hash, sequence_type, sequence, name, file_path, "length") values ("start-node-yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy", "OTHER", "start-node-yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy", "", "", 64), ("end-node-zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz", "OTHER", "end-node-zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz", "", "", 64);
+INSERT INTO gen_metadata (db_uuid) values (lower(
+    hex(randomblob(4)) || '-' || hex(randomblob(2)) || '-' || '4' ||
```
Comment (dkhofer) [inline migrations/01-initial/up.sql:101]: Is this like a standard UUID format?

File: src/operation_management.rs [hunk 2]
```diff
@@ -85,6 +89,29 @@ pub fn revert_changeset(conn: &Connection, op_id: i32) {
     conn.pragma_update(None, "foreign_keys", "1").unwrap();
 }
 
+pub fn move_to(conn: &Connection, op_id: i32) {
```
Comment (dkhofer) [inline src/operation_management.rs:92]: I don't see any way around this, but I hope we don't end up reinventing a bunch of git internals

**Comment by dkhofer** [comment]
LGTM

**Review by dkhofer — APPROVED** [review]

File: migrations/01-initial/up.sql (hunk 1 above)
Comment (Chris7) [reply migrations/01-initial/up.sql:101]: yeah, we can discuss if we should make this something simpler like the local db path and checksum it or something.

**Pull request merged**